subprojects/platform-base/src/main/java/org/gradle/api/BuildableComponentSpec.java

```diff
@@ -18,6 +18,8 @@
 
 import org.gradle.platform.base.ComponentSpec;
 
+import javax.annotation.Nullable;
```

Comment:
No, imports take precedence. That's what IntelliJ is reporting, and I also ran this test:

```groovy
    def "test"() {
        def thing = BuildableComponentSpec.declaredMethods.find { it.name == "getBuildTask" }
        expect:
        thing.annotations.each {
            println "$thing has annotation: $it"
        }
    }
```

...which printed:

```text
public abstract org.gradle.api.Task org.gradle.api.BuildableComponentSpec.getBuildTask() has annotation: @javax.annotation.Nullable()
```